Subject:       Print paths for legacy TBB or oneTBB
Patch Author:  Orson Teodoro <orsonteodoro@hotmail.com>
Date:          Tue Jun 29 11:13:33 AM PDT 2021 (Unix timestamp: 1624990413)

Copyright notice for cmake/FindTBB.cmake:

## Copyright 2009-2021 Intel Corporation
## SPDX-License-Identifier: Apache-2.0

diff -urp oidn-1.4.1.orig/cmake/FindTBB.cmake oidn-1.4.1/cmake/FindTBB.cmake
--- oidn-1.4.1.orig/cmake/FindTBB.cmake	2021-06-22 15:31:19.000000000 -0700
+++ oidn-1.4.1/cmake/FindTBB.cmake	2021-06-29 11:12:27.771813945 -0700
@@ -36,6 +36,8 @@
 #
 #===============================================================================
 
+# oteodoro - print out tbb paths to show preference towards legacy TBB or oneTBB
+
 # We use INTERFACE libraries, which are only supported in 3.x
 cmake_minimum_required(VERSION 3.1)
 
@@ -371,6 +373,7 @@ function(rk_tbb_find_library COMPONENT_N
     NAMES ${LIB_NAME}
     PATHS ${LIB_PATHS}
     NO_DEFAULT_PATH)
+  message(STATUS "TBB lib: ${${LIB_VAR}}")						# oteodoro - added line
 
   # Hide this variable if we found something, otherwise display it for
   # easy override.
@@ -490,3 +493,4 @@ endif()
 
 set(TBB_FOUND TRUE)
 set(TBB_INCLUDE_DIRS "${TBB_INCLUDE_DIR}")
+message(STATUS "TBB include dir: ${TBB_INCLUDE_DIR}")					# oteodoro - added line
